app/modules/ads/bidders-delay.js

```diff
@@ -36,6 +36,8 @@ export default {
         resolvePromise();
         resolvePromise = null;
       }
+
+      this.biddersCalled = true;
```

Comment:
this `this` is confusing. It operates on `Ads` module like on itself

Reply:
Yeah, that's true... \/\/\/